app/src/app.gleam

```diff
@@ -19,9 +20,9 @@ pub type Model {
   Model(rendered_lustre: String, copy_button_text: String)
 }
 
-const copy_button_default_text = "Copy 🧟"
+const copy_button_default_text = "Copy"
 
-const copy_button_copied_text = "Copied 🥰"
+const copy_button_copied_text = "Copied"
```

Comment:
Why have these been removed?

Reply:
I put in an SVG icon for the copy, which is perhaps a bit less joyous but is a bit more semantically clear

Reply:
One change per PR please, revert this